src/puppetlabs/ezbake/core.clj

```diff
@@ -406,7 +426,8 @@ Dependency tree:
                                                           :main-namespace
                                                           "puppetlabs.trapperkeeper.main")
          :java-args                 (get-local-ezbake-var lein-project :java-args
-                                                          "-Xmx192m")}))))
+                                                          "-Xmx192m")
+         :bootstrap-source          (get-local-ezbake-var lein-project :bootstrap-source :file)}))))
```

Comment:
That's true for most of the other config settings already 😢 . java-args, terminus-files...